lab_3_keywords_textrank/main.py

```diff
@@ -385,7 +453,17 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        if vertex1 not in self._edges:
```

Comment:
create a separate function that has 2 arguments and use twice

Reply:
done